eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/sorted/immutable/ImmutableSortedBagImpl.java

```diff
@@ -792,6 +793,12 @@ public int hashCode()
         return counter.getCount();
     }
 
+    @Override
+    public RichIterable<T> distinctView()
+    {
+        return new ImmutableSortedSetFromArrayBagAdapter(this.comparator, this.elements);
```

Comment:
IntelliJ inspections 4 build is failing on Raw use of parameterized class here

Reply:
Yep my bad. Anyway, I'm now convinced it's a bad idea to add those new classes since one is just going to reimplement the same thing as `ImmutableTreeSet` and the other is no better than just `ArrayAdapter`.

Reply:
@victornoel Makes sense.